app/models/area.rb

```diff
@@ -4,12 +4,14 @@ class Area < ActiveRecord::Base
 
   class << self
     def for_code(area_code)
+      return nil unless area_code.present?
```

Comment:
also, I typically prefer `if object.blank?` over `unless present?` since I think it reads better. thoughts on that?